de_DE/LC_MESSAGES/discordbot.po

```diff
@@ -49,47 +49,47 @@ msgstr "%d Suchergebnisse:"
 
 # SongName by BandName
 msgid "MUSICMATCH_TITLE_BY_ARTIST"
-msgstr "%s von %s"
+msgstr "%s - %s"
 
 msgid "PATRON_TICKET_SUCCESS"
-msgstr "You received a %s Sudomemo Patron Ticket!"
+msgstr "Du hast ein %s Sudomemo Patron Ticket erhalten."
 
 msgid "REJECT_ACCOUNT_ALREADY_LINKED"
-msgstr "Du hast dein Sudomemo-Konto bereits verbunden!"
+msgstr "Du bist bereits in Sudomemo angemeldet."
 
 msgid "REJECT_BOOST_TICKET_COOLDOWN"
-msgstr "Du hast bereits ein Nitro Booster-Ticket in den letzten 30 Tagen erhalten! (%s)"
+msgstr "Du hast bereits ein Nitro Booster-Ticket in den letzten 30 Tagen erhalten, (%s)."
 
 msgid "REJECT_FSID_ENOENT"
-msgstr "Für diese Flipnote Studio ID wurde kein Benutzer gefunden."
+msgstr "Diese Flipnote Studio ID (FSID) existert nicht."
 
 msgid "REJECT_GENERIC_ERROR"
-msgstr "Bei dieser Aktion trat ein Problem auf. Sorry!"
+msgstr "Ein Fehler ist aufgetreten. Bitte versuche es in 5 Minuten noch einmal."
```

Comment:
No duration should be specified. This is a generic error so the message should also be generic. Also this removes the exclamation mark.